test/metabase/test/data/impl.clj

```diff
@@ -265,9 +265,10 @@
 
 (defn- copy-db-tables! [old-db-id new-db-id]
   (let [old-tables    (t2/select Table :db_id old-db-id, :active true, {:order-by [[:id :asc]]})
-        new-table-ids (t2/insert-returning-pks! Table
-                        (for [table old-tables]
-                          (-> table (dissoc :id) (assoc :db_id new-db-id))))]
+        new-table-ids (sort ; insert-returning-pks! doesn't order by pk or by insertion order
```

Comment:
I would explicitly say that ordering by pk recovers the insertion order

Reply:
yep. went with `sorting by PK recovers the insertion order, because insert-returning-pks! doesn't guarantee this`